src/sentry/sentry_metrics/metrics_wrapper.py

```diff
@@ -29,7 +28,7 @@ def __merge_tags(self, tags: Optional[Tags]) -> Optional[Tags]:
         elif tags is None:
             return self.__tags
         else:
-            return ChainMap(tags, self.__tags)
+            return {**self.__tags, **tags}
```

Comment:
`ChainMap` is a `MutableMapping` and requires all of its inputs to be `MutableMapping`s (it should really be polymophically typed but iirc that requires HKTs which don't exist in mypy(!)) -- a dict splat works just as well (and likely is faster in the end anyway since all of the attributes get accessed)